### ISSUE TITLE
peerDependencies do not match devDependencies

### ISSUE THREAD

**Issue by kburson** [rolaveric/karma-systemjs#2]
In devDependencies you all systemjs:"^0.11.2" (which would allow my project with systemjs:"0.14.1"),
but in your peerDependencies (which is deprecated and will be removed from npm 3.0) you list it as "systemjs":~0.11.2", which means I can only update "0.11.x" as opposed to "0.x.x".  Also, the traceur versions do not match up.

Please modify this by updating peerDependency to use '^', or remove peerDependencies all-together.

```json
  "devDependencies": {
    "es6-module-loader": "^0.11.2",
   ...
    "systemjs": "^0.11.2",
    "traceur": "0.0.81"
  },
  "peerDependencies": {
    "systemjs": "~0.11.2",
    "es6-module-loader": "~0.11.2",
    "traceur": "~0.0.79"
  }
```

thanks,

**Comment by rolaveric** [comment]
Thanks for the heads up and the PR.  Merged in ab1df4a31c575eb34e40c33d0fac125f53aab4c8
I'll add doco stating that 'traceur', 'es6-module-loader', and 'systemjs' need to be installed independently, and add error handling around their 'require()' calls, similar to how karma does for plugins.

**Comment by rolaveric** [comment]
Updated doco and error messaging in 1f2b8b7e29411a00fb081dbadf5ccf0578ea5f3c
Published v0.1.3 to npm
Closing
